sdk/README.md

```diff
@@ -111,21 +111,38 @@ If you cloned the `kfp-tekton` project, you can find example pipelines in the
 After compiling the `sdk/python/tests/compiler/testdata/parallel_join.py` DSL script
 in the step above, we need to deploy the generated Tekton YAML to our Kubernetes
 cluster with `kubectl`. The Tekton server will automatically start a pipeline run
-for which we can follow the logs using the `tkn` CLI:
+for which we can follow the logs using the `tkn` CLI. Here we have to deploy the
+pipeline in the kubeflow namespace because all the pipelines with metadata
+tracking have to rely on the minio credential in the kubeflow namespace:
```

Comment:
```suggestion
and artifacts tracking  rely on the minio object storage credentials in the kubeflow namespace.
```

Reply:
Additionally - possible for someone to use S3 backed object storage credentials in another namespace and launch pipelines there if they need artifacts and metadata to be stored there?

Reply:
Yes, as long as they provide the credential in that namespace. For the recent version of Argo, they are able to inject the default S3 credential as string to the pod's env variables. Since Tekton doesn't have the concept of artifact yet, we might have to create our own webhook controller to do this. Another alternative way is to create our custom profile controller to create the default secret on the user namespace.